apps/SageAccountWeb/src/AppInitializer.tsx

```diff
@@ -16,56 +11,13 @@ import { AppContextProvider } from './AppContext'
 import { useSourceApp } from './components/useSourceApp'
 
 function AppInitializer(props: { children?: React.ReactNode }) {
-  const [appId, setAppId] = useState<string>()
   const [redirectURL, setRedirectURL] = useState<string>()
-  const [theme, setTheme] = useState<Theme>(
-    createTheme(
-      SynapseTheme.defaultMuiThemeOptions,
-      sageAccountWebThemeOverrides,
-    ),
-  )
+
   const [signedToken, setSignedToken] = useState<
     SignedTokenInterface | undefined
   >()
-  const { currentSourceAppNameState } = useLastLoginInfoState()
   const isFramed = useFramebuster()
-
-  useEffect(() => {
-    const searchParamAppId = getSearchParam('appId')
-    const localStorageAppId = localStorage.getItem('sourceAppId')
-    if (searchParamAppId) {
-      localStorage.setItem('sourceAppId', searchParamAppId)
-      setAppId(searchParamAppId)
-    } else if (localStorageAppId) {
-      setAppId(localStorageAppId)
-    } else {
-      // fallback to Sage Bionetworks
-      localStorage.setItem('sourceAppId', 'SAGE')
-      setAppId('SAGE')
-    }
-  }, [])
-
-  const sourceApp = useSourceApp(appId)
-
-  useEffect(() => {
-    if (sourceApp?.friendlyName) {
-      currentSourceAppNameState.set(sourceApp.friendlyName)
-    }
-  }, [currentSourceAppNameState, sourceApp?.friendlyName])
```

Comment:
This effect set a localStorage value that we would use to inform you you could use your Sage account thaat you used to log into a different app. Moved to `SourceAppProvider`